src/common/const.ts

```diff
@@ -122,6 +122,7 @@ export const FIXED_DOMAIN_ICONS = {
 };
 
 export const FIXED_DEVICE_CLASS_ICONS = {
+  accumulated_precipitation: mdiWater,
```

Comment:
May be `mdiWeatherRainy` ?
![image](https://user-images.githubusercontent.com/5878303/198680297-e6de9ec0-1aa7-4b1c-98c8-f042b95f666d.png)